vmaas_sync/system_culling_test.go

```diff
@@ -82,7 +82,7 @@ func TestMarkSystemsStale(t *testing.T) {
 	for i := range systems {
 		assert.NoError(t, database.Db.Save(&systems[i]).Error)
 	}
-	nMarked, err := markSystemsStale(database.Db)
+	nMarked, err := markSystemsStale(database.Db, 500)
```

Comment:
This doesn't test that limit works properly. We should set lower limit and see that some systems stayed unmarked.